metaflow/meta_files.py

```diff
@@ -0,0 +1,132 @@
+import json
+import os
+
+from enum import Enum
+from typing import Any, Dict, Optional, Union
+
+from .util import get_metaflow_root
+
+_info_file_content = None
+_info_file_present = None
+_included_dist_info = None
+_included_dist_present = None
+
+# Ideally these would be in package/mfenv.py but that would cause imports to fail so
+# moving here. The reason is that this is needed to read extension information which needs
+# to happen before mfenv gets packaged.
+
+MFENV_DIR = (
+    ".mfenv"  # Directory containing "system" code (metaflow and user dependencies)
+)
+MFCONF_DIR = ".mfconf"  # Directory containing Metaflow's configuration files
+MFENV_MARKER = (
+    ".mfenv_install"  # Special file containing metadata about how Metaflow is packaged
+)
+
+
+class MetaFile(Enum):
+    INFO_FILE = "INFO"
+    CONFIG_FILE = "CONFIG_PARAMETERS"
+    INCLUDED_DIST_INFO = "INCLUDED_DIST_INFO"
```

Comment:
what is `INCLUDED_DIST_INFO` - hard to understand what this implies - is there an `EXCLUDED_DIST_INFO` too somewhere?

Reply:
I can clarify: it means the information about any distribution that is included. So no, there is no excluded one. I can just hame it DIST_INFO.